src/app/api/quality_matrix/models.py

```diff
@@ -1,18 +1,38 @@
+from dataclasses import dataclass
 from enum import Enum
-from typing import Union
+from typing import Optional
 
 from pydantic import BaseModel, Field
 from sqlalchemy import JSON, Column, Integer, Text
 from sqlalchemy.orm import declarative_base
 
 
-class ColumnOutputModel(BaseModel):
+@dataclass(frozen=True)
+class QualityOutput:
+    metadatum: str
```

Comment:
eventually rename to "column_name" or "row_name" (not sure what it actually is). See also other comment on [src/app/api/quality_matrix/collections.py](https://github.com/openeduhub/metaqs-main/pull/70/files#diff-546186f01cc6a2e5c28c419acb25b21182c20858941c803113776099ccf56438)

Reply:
as there is a `columns` dict below, I assume it is `row_name` or just `row`

Reply:
Renamed to row_header. Thanks for pointing this out - I was taking it for granted.